src/net-ipfs-engine/Catalyst.Ipfs.Http.Server/HttpApi/V0/PubSubController.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Catalyst.Ipfs.Core.CoreApi;
+using Lib.P2P.PubSub;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Catalyst.Ipfs.Http.Server.HttpApi.V0
+{
+    /// <summary>
+    ///   A list of topics.
+    /// </summary>
+    public class PubsubTopicsDto
+    {
+        /// <summary>
+        ///   A list of topics.
+        /// </summary>
+        public IEnumerable<string> Strings;
+    }
+
+    /// <summary>
+    ///   A list of peers.
+    /// </summary>
+    public class PubsubPeersDto
+    {
+        /// <summary>
+        ///   A list of peer IDs.
+        /// </summary>
+        public IEnumerable<string> Strings;
+    }
+
+    /// <summary>
+    ///   A published message.
+    /// </summary>
+    public class MessageDto
+    {
+        /// <summary>
+        ///   The base-64 encoding of the author's peer id.
+        /// </summary>
+        public string from;
+
+        /// <summary>
+        ///   The base-64 encoding of the author's unique sequence number.
+        /// </summary>
+        public string seqno;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)